doc/support_matrix.yml

```diff
@@ -22,36 +22,62 @@ support_keys:
 functionalities:
   - name: gpio
     title: GPIO
+    # Name(s) of the laze app(s) (as defined in the app's `laze.yml`)
+    # that must be built for the functionality to be marked
+    # as supported for a specific piece of hardware.
+    # Can be tests or examples.
+    support_criteria:
+      - tests_gpio
```

Comment:
I think it would be nifty if not only applications but also modules could be used here.